src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " Excavator started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getName() + " Excavator finished my work.");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It's good practice to keep the messages consistent. Consider changing the message to 'Excavator has finished its work.' to clearly indicate the type of machine.